src/helix-ui/elements/HXDivElement.less

```diff
@@ -1,3 +1,5 @@
+@import (reference) './HXElement';
```

Comment:
Please remove `(reference)`. We need the CSS from `HXElement.less` to load inline for this file, but `(reference)` prevents that from happening. This is why it seems like we need the `box-sizing` property in `pills.less` below.

Reply:
Updated with above feedback.